tests/restv1/PayPal/CartUpdateCest.php

```diff
@@ -66,12 +70,7 @@ public function should_allow_updating_cart_for_post( Restv1Tester $I ) {
 			],
 		], $response['tickets'] );
 
-		$I->assertContains(
-			'?tribe_tickets_redirect_to=https%3A%2F%2Fwww.sandbox.paypal.com%2Fcgi-bin%2Fwebscr%2F_cart'
-				. '%3Fcmd%3D_cart'
-				. '%26business%3Dmerchant%2540example.com'
-				. '%26bn%3DModernTribe_SP'
-			, $response['cart_url'] );
+		$I->assertContains( '?tribe_tickets_redirect_to=https%3A%2F%2Fwww.sandbox.paypal.com%2Fcgi-bin%2Fwebscr%2F_cart' . '%3Fcmd%3D_cart' . '%26business%3Dmerchant%2540example.com' . '%26bn%3DModernTribe_SP', $response['cart_url'] );
```

Comment:
did you want to format the code the way it was, so it is easier to read?

Reply:
Fixed ✅